doc/source/reference/credits.rst

```diff
@@ -287,6 +287,11 @@ Mike Henry
 * Fixed syntax in freud-examples notebooks for v2.0.
 * Updated documentation links
 
+Michael Stryk
+
+* Added short examples into Cluster, Density, Environment, and Order Modules.
+* Updated old API in freud ovito-scripts.
```

Comment:
I'd remove this line (and maybe add a credits file if one doesn't exists to the ovito-scripts repo so we can keep track of who did what there).